forem/forem
# Badges design fixes as suggested by Anuj

## What type of PR is this? (check all applicable)


## Description

Following [changes](https://www.figma.com/file/ecvlal5QFT01C1idbO3Ljx/Badges-%F0%9F%8E%96?node-id=801%3A860) were suggested by @anujbhavsar96 for badges page. This PR applies those changes and also make sure that other UI related things are also taken care of.

<img width="750" alt="Screenshot 2022-10-27 at 4 31 00 PM" src="https://user-images.githubusercontent.com/9396084/198288336-7b4d1923-43f7-4515-b529-684f0266f52d.png">
<img width="750" alt="Screenshot 2022-10-27 at 4 31 12 PM" src="https://user-images.githubusercontent.com/9396084/198288363-915a6277-5fb0-4d81-ae4f-ea9a8c5aaaeb.png">
<img width="750" alt="Screenshot 2022-10-27 at 4 31 34 PM" src="https://user-images.githubusercontent.com/9396084/198288377-cd99813c-85c4-47b1-bac6-e6fb81b9d3f0.png">
<img width="250" alt="Screenshot 2022-10-27 at 4 31 40 PM" src="https://user-images.githubusercontent.com/9396084/198288379-30229c2b-2bee-4621-9981-be59ce6296f8.png">

### How to test this PR?

- Visit localhost:3000/badges and also click on any badge to see results
- Also, visit any profile with awarded badges and click on any badge
- Change theme to dark-mode in `Settings` and again visit badges page
- Check badges page across browsers
- Check badges page across screen-sizes

## Related Tickets & Documents



- Related Issue #18647 
- Closes #18647

## QA Instructions, Screenshots, Recordings

### Chrome Badge Page -- across sizes

<img width="800" alt="Screenshot 2022-11-02 at 12 52 27 AM" src="https://user-images.githubusercontent.com/9396084/199323072-68e14029-fc0c-4572-ba16-69d13e4dd0f7.png">

<img width="750" alt="Screenshot 2022-11-02 at 12 52 59 AM" src="https://user-images.githubusercontent.com/9396084/199323084-03b9832f-38e6-45c3-9e84-8f034f7f81f3.png">

<img width="375" alt="Screenshot 2022-11-02 at 12 53 18 AM" src="https://user-images.githubusercontent.com/9396084/199323087-dcf928e3-f4af-4a33-b437-ea75c731c19f.png">

### Chrome Badge Modal -- across various cases


<img width="800" alt="Screenshot 2022-11-02 at 12 52 35 AM" src="https://user-images.githubusercontent.com/9396084/199323204-eb6d1bc0-c899-4bcd-b75b-8bc3739fd814.png">

<img width="800" alt="Screenshot 2022-11-02 at 12 52 41 AM" src="https://user-images.githubusercontent.com/9396084/199323213-4d3f78e4-4baf-4a0c-a0f1-b93892aa51d4.png">

<img width="300" alt="Screenshot 2022-11-02 at 12 53 37 AM" src="https://user-images.githubusercontent.com/9396084/199323217-8e859783-4a7c-4757-b620-f6d343a5a4d3.png"> <img width="300" alt="Screenshot 2022-11-02 at 12 53 50 AM" src="https://user-images.githubusercontent.com/9396084/199323220-6924185c-120a-4a4d-9c9d-4726b6e5989f.png">

### Dark-mode theme
 
<img width="800" alt="Screenshot 2022-11-02 at 12 51 50 AM" src="https://user-images.githubusercontent.com/9396084/199323425-9a747bd7-860f-4362-bac9-1089339474be.png">
<img width="800" alt="Screenshot 2022-11-02 at 12 51 57 AM" src="https://user-images.githubusercontent.com/9396084/199323445-4225605f-3b0e-4793-9dcf-0349e568b4d3.png">


### Other Browsers

Firefox & Safari 15+ screenshots have not been shared as they look exactly same as that of Chrome.

Note: Safari 14 and below does not support `aspect-ratio` as a result showing perfect square for this design was not possible. But we do make sure that the UI does look great even if it's not square. A similar issue has already been filed #18452 

<img width="800" alt="Screenshot 2022-11-02 at 1 02 14 AM" src="https://user-images.githubusercontent.com/9396084/199323905-0ac7a21b-0c5f-4c95-9477-7371c07bc1f0.png">
<img width="575" alt="Screenshot 2022-11-02 at 1 02 26 AM" src="https://user-images.githubusercontent.com/9396084/199323911-28944ede-8397-4441-980e-d7b7d169072e.png">



### UI accessibility concerns?

No

## Added/updated tests?

- [x] No, and this is why: only design changes

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
